### PR TITLE
Oleksii/OB_178_pig_latin

### PR DESCRIPTION
**Pull request by alexone55**

### CODE REVIEW

File: Alexey/Text/pig_latin/test/test_pig_latin.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import unittest
+from Alexey.Text.pig_latin.main.pig_latin import translater
+
+
+class TestPigLatin(unittest.TestCase):
+
+    def test_send_none_and_expect_attribute_error(self):
+        with self.assertRaises(AttributeError) as context:
```
Comment (glock3) [inline Alexey/Text/pig_latin/test/test_pig_latin.py:8]: You have checked exception twice here. Please remove one check

File: Alexey/Text/pig_latin/test/test_pig_latin.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        exception_message = str(context.exception)
+        self.assertEqual('AttributeError', exception_message)
+
+    def test_send_sentence_and_expect_pig_latin_translate(self):
+        expect_set = "ityay isyay igpay atinlay entencesay esttay"
+        input_sentence = translater(str("It is pig latin sentence test"))
+        self.assertEqual(expect_set, input_sentence)
+
+    def test_send_number_and_expect_attribute_error(self):
+        with self.assertRaises(AttributeError) as context:
```
Comment (glock3) [inline Alexey/Text/pig_latin/test/test_pig_latin.py]: Same as above

**Review by glock3 — CHANGES_REQUESTED** [review]

File: Alexey/Text/pig_latin/test/test_pig_latin.py (hunk 1 above)
Comment (alexone55) [reply Alexey/Text/pig_latin/test/test_pig_latin.py:8]: done

Comment (glock3) [reply Alexey/Text/pig_latin/test/test_pig_latin.py:8]: Nope, wrote in pm

**Review by glock3 — CHANGES_REQUESTED** [review]

**Review by glock3 — APPROVED** [review]

**Pull request merged**